File: src/features/auth/dto/password-rules.ts
export const STRONG_PASSWORD_REGEX = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^\w\s]).{12,}$/;
export const STRONG_PASSWORD_MESSAGE =
  'Password must be at least 12 characters and include uppercase, lowercase, number, and symbol.';

File: spring-boot-api/src/main/java/com/mytegroup/api/dto/auth/PasswordRules.java
package com.mytegroup.api.dto.auth;

public class PasswordRules {
    public static final String STRONG_PASSWORD_REGEX = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[^\\w\\s]).{12,}$";
    public static final String STRONG_PASSWORD_MESSAGE = "Password must be at least 12 characters and include uppercase, lowercase, number, and symbol.";
}



